showcase/src/main/kotlin/com/recco/showcase/data/room/PaletteDao.kt

```diff
@@ -0,0 +1,19 @@
+package com.recco.showcase.data.room
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+import androidx.room.Update
+import com.recco.showcase.data.entities.ShowcasePaletteEntity
+
+@Dao
+interface PaletteDao {
+    @Query("SELECT * FROM palette")
+    suspend fun get(): List<ShowcasePaletteEntity>
```

Comment:
it seems that should be renamed to `getAll()`

Reply:
done 👍